cli/cmd/create.go

```diff
@@ -88,14 +89,11 @@ func (cc *CreateCommand) bindFlagsFunction() func(commandFlags map[string]func()
 				}
 			} else {
 				var key string
-				cmd.PersistentFlags().StringVar(&key, flagName, "", flagDesc)
+				cmd.PersistentFlags().StringVar(&key, flagName, flag.FlagDefault(), flagDesc)
 				commandFlags[flagName] = func() string {
 					return key
 				}
 			}
-			if flag.FlagRequired() {
```

Comment:
Why delete this line

Reply:
At line 80-82, it is already exists......